src/main/java/com/oltpbenchmark/DBWorkload.java

```diff
@@ -81,11 +83,17 @@ public static void main(String[] args) throws Exception {
         }
 
 
-        // Seconds
-        int intervalMonitor = 0;
+        // Monitoring setup.
+        ImmutableMonitorInfo.Builder builder = ImmutableMonitorInfo.builder();
         if (argsLine.hasOption("im")) {
-            intervalMonitor = Integer.parseInt(argsLine.getOptionValue("im"));
+            builder.monitoringInterval(Integer.parseInt(argsLine.getOptionValue("im")));
+        }
+        if (argsLine.hasOption("mt")) {
+            if (argsLine.getOptionValue("mt").equals("advanced")) {
+                builder.monitoringType(MonitorInfo.MonitoringType.ADVANCED);
+            }
         }
```

Comment:
Maybe restructure this a tad so that if `-mt` is passed and it's not one of the known formats we throw an error about invalid cli args.